manifests/profile/hathitrust/cron/mdp_misc.pp

```diff
@@ -16,16 +16,18 @@
   String $sdr_data_root = '/sdr1',
   String $home = '/htapps/babel/mdp-misc',
   String $catalog_home = '/htapps/catalog/web',
-  Integer $mdp_sessions_minute = 5
+  Integer $mdp_sessions_minute = 5,
+  $sdr_environment = [
```

Comment:
Do we want this in the parameters section? This would allow us to override the defaults; not sure if/when we would want to do that.

Reply:
Let's leave it alone for the moment. I need to re-jigger those scripts anyway so they're less...bad.

Reply:
:heavy_check_mark: